packages/airnode-admin/docker/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+ADMIN_BINARY="/usr/local/bin/airnode-admin"
+
+node ${ADMIN_BINARY} "$@"
```

Comment:
Do we even need this file? You can call the binary directly from `ENTRYPOINT` and arguments are passed automatically by Docker.